charts/apisix/Chart.yaml

```diff
@@ -31,7 +31,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.4.0
+version: 1.4.1
```

Comment:
We can leave this unchanged for now, right? This will trigger the release of a new version.
I want to check the recent modifications and see if they include any changes.

Reply:
Hello @tao12345666333, I made a rollback on the chart semver as asked.